index.js

```diff
@@ -38,6 +39,7 @@ const errorHandler = (e, isFatal) => {
         `
     );
   } else {
+    // eslint-disable-next-line no-console
```

Comment:
Can't we enable this only in dev mode?

Reply:
> Can't we enable this only in dev mode?

I dont't think to get your suggestion, what do you mean by _enable this_ in this specific point?